index.html

```diff
@@ -33,8 +150,63 @@ <h2 class="blue-text">LET'S CONNECT</h2>
       <li><a href="#"><img src="./img/github.png" alt="Github"></a></li>
       <li><a href="#"><img src="./img/angel-list.png" alt="Angel-list"></a></li>
     </ul>
+    <button class="big-button" type="submit" name="submit">Get My Resume</button>
+    <ul class="frame flex-button">
+      <li>
+        <h2>Languages</h2>
+      </li>
+      <li><img src="./img/Vector.png" alt="lang"></li>
+    </ul>
+
+    <div class="buttons">
+      <div class="button-fin">
+        <img src="./img/js.svg" alt="js-button">
+        <p>JavaScript</p>
+      </div>
+    </div>
+
+    <div class="buttons">
+      <div class="button-fin">
+        <img src="./img/html.svg" alt="html-button">
+        <p>HTML</p>
+      </div>
+    </div>
+
+    <div class="buttons">
+      <div class="button-fin">
+        <img src="./img/css.svg" alt="css-button">
+        <p>CSS</p>
```

Comment:
The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`.

Reply:
Fixed.